boards/arm/96b_carbon/doc/96b_carbon.rst

```diff
@@ -10,6 +10,39 @@ Zephyr applications use the 96b_carbon configuration to run on the 96Boards
 Carbon hardware. It is based on the STMicroelectronics STM32F401RET Cortex-M4
 CPU and also contains a nRF51832 chip connected over SPI for BLE connectivity.
 
+.. note::
+
+   The nRF51832 chip on this board is supported by Zephyr using
```

Comment:
That is already written above:

"It is based on the STMicroelectronics STM32F401RET Cortex-M4  CPU and also contains a nRF51832 chip connected over SPI for BLE connectivity."